src/DatePicker/DatePicker.js

```diff
@@ -37,6 +37,16 @@ class DatePicker extends Component {
         this.popoverRef = React.createRef();
     }
 
+    componentDidMount() {
+        const { disableStyles } = this.props;
+
+        if (!disableStyles && this._showTodayFooter()) {
+            //below styles are needed for footer styling
+            require('fundamental-styles/dist/dialog.css');
+            require('fundamental-styles/dist/bar.css');
```

Comment:
I think this is the old way of including styles, now they're expected to be imported with the rest of the imports without conditionals.